src/lib/backend-api.ts

```diff
@@ -222,3 +222,8 @@ export const getKey = async (
   const response = await request(`${getBackendApiUrl()}/btc/key/${keyId}${includePrivate ? `?includePrivate=${includePrivate}` : ''}`, options)
   return response.data
 }
+
+export const getRecommendedFee = async () : Promise<GetFeeRate> => {
```

Comment:
to zwraca wszystkie fees, nie tylko recommended, to się chyba powinno nazywać `getFees`? Albo, `getRecommendedFee`, i zwracac tylko pole recommended?